### PR TITLE
Now Sawgger Generator Describe CRUD Actions Only IF Action Exist

### PR DESCRIPTION
**Pull request by DenisPopov15**

### CODE REVIEW

File: lib/swagger/generator.rb [hunk 1]
```diff
@@ -28,8 +43,8 @@ def resource_name
       @resource_name ||= collection_name.singularize
     end
 
-    def generate_swagger_schemas
-      name = resource_name
+    def generate_swagger_schemas(name=false)
```
Comment (alexkravets) [inline lib/swagger/generator.rb:71]: please remove '=false' - no need

File: lib/swagger/generator.rb [hunk 2]
```diff
@@ -54,126 +69,161 @@ def generate_swagger_schemas
           end
         end
       end
+
+      # For Input as Param in UI
```
Comment (alexkravets) [inline lib/swagger/generator.rb]: this comment is not clear

File: lib/swagger/generator.rb [hunk 3]
```diff
@@ -20,6 +21,20 @@ def generate_swagger
       generate_swagger_paths
     end
 
+    def fetch_available_actions
```
Comment (alexkravets) [inline lib/swagger/generator.rb]: fetch_mounted_actions

**Comment by alexkravets** [comment]
Please bump gem version in versio.rb. comments are added.

File: lib/swagger/generator.rb (hunk 1 above)
Comment (DenisPopov15) [inline lib/swagger/generator.rb:71]: Why? `generate_swagger_schemas` can be called without any arguments :https://github.com/slate-studio/mongoid-openapi/blob/4e77a64c3ae503bb5674128fbf3e71794e988e42/lib/swagger/generator.rb#L20. Without `=false` will be error "ArgumentError (wrong number of arguments (given 0, expected 1)):"

Comment (alexkravets) [inline lib/swagger/generator.rb:71]: @DenisPopov15 yes, sorry. You are right, I've messed this up with CoffeeScript. Still, please change default `false` value to default `nil`.

**Comment by DenisPopov15** [comment]
@alexkravets updated

**Pull request merged**